packages/portfolio-contract/tools/portfolio-actors.ts

```diff
@@ -344,11 +344,18 @@ export const planDepositTransfers = (
   return transfers;
 };
 
-export const planTransfer = (
+type GasEstimator = (chainName: AxelarChain) => {
```

Comment:
a GasEstimator is a function that takes a chainName and gives back a thing with methods? maybe call it...

```suggestion
type MakeGasEstimator = (chainName: AxelarChain) => {
```